tasks/components/layout/OrganizationDetails.tsx

```diff
@@ -70,7 +73,8 @@ export const OrganizationDetail = ({
   } = useContext(OrganizationContext)
 
   const { signOut, organizations } = useAuth()
-  const isCreatingNewOrganization = contextState.organizationId === '' || !organizations.includes(contextState.organizationId)
+  const { fakeTokenEnable } = getConfig()
+  const isCreatingNewOrganization = contextState.organizationId === '' || !(fakeTokenEnable || organizations.includes(contextState.organizationId))
```

Comment:
this boolean shouldn't be touched as it determines the state of creating and loading an organization.

For any change regarding showing a component based on organization claims do it seperatly?
e.g. if your intent is to show the create screen if the user doesn't has access to the organization, do it in the organization page and not in this view
```suggestion
  const isCreatingNewOrganization = contextState.organizationId === ''
```